cuckoo/query.py

```diff
@@ -315,6 +317,13 @@ def _build_many_models(self):
         for data in data_list:
             yield self.model(**data)
 
+    def _build_many_models_stream(self):
+        for item in ijson.items(
+            IterByteIO(self._root._response.iter_bytes()),
+            f"data.{self._query_alias}.item",
+        ):
+            yield self.model(**item)
```

Comment:
part 4 of the feature: once we have a streaming response, we call `iter_bytes()`, which returns a byte generator. since `ijso` wants a file-like object, we convert the generator with a custom thin wrapper  `IterBytesIO`